docs/worker-node/install-singularity.md

```diff
@@ -139,12 +145,12 @@ UID        PID  PPID  C STIME TTY          TIME CMD
 user         1     0  0 21:34 ?        00:00:00 ps -ef
 ```
 
-!!! tip
-    The remainder of this document pertains to the un-privileged
-    (non-`setuid`) mode of singularity.
+Unprivileged Singularity
+------------------------
+
+The instructions in this section are for installing singularity with non-setuid executables.
```

Comment:
This isn't actually true -- the instructions are not for installing anything, they for enabling singularity to run with non-setuid executables.

Reply:
In fact I think it would be good to have a second sentence to transition to the concept of "via OASIS", by saying that a non-setuid distribution of singularity is already installed in OASIS and these instructions enable it to run.  Otherwise "via OASIS" is kind of mysterious, unless people remember that all way back from the 3rd intro paragraph at the top.  There the term is actually "cvmfs".